From 0a5b3b955cff71f53bb66047b612fb142b1be309 Mon Sep 17 00:00:00 2001
From: Patric Stout <truebrain@openttd.org>
Date: Sat, 16 Mar 2019 10:00:58 +0100
Subject: [PATCH 0473/1053] Fix: [AzurePipelines] vcpkg is now preinstalled on
 Windows images

So instead of integrating our own, we only copy our precompiled
binaries into the right folder.
---
 azure-pipelines/templates/windows-dependencies.yml | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/azure-pipelines/templates/windows-dependencies.yml b/azure-pipelines/templates/windows-dependencies.yml
index c6fc6e407c..72368b9d71 100644
--- a/azure-pipelines/templates/windows-dependencies.yml
+++ b/azure-pipelines/templates/windows-dependencies.yml
@@ -1,10 +1,14 @@
 steps:
 - bash: |
     set -ex
+
     curl -L https://github.com/OpenTTD/CompileFarm/releases/download/latest/windows-dependencies.zip > windows-dependencies.zip
     unzip windows-dependencies.zip
     rm -f windows-dependencies.zip
-  displayName: 'Download dependencies'
-  workingDirectory: $(Build.ArtifactStagingDirectory)
-- script: $(Build.ArtifactStagingDirectory)\windows-dependencies\vcpkg.exe integrate install
+
+    mv windows-dependencies/installed /c/vcpkg/
+    rm -rf windows-dependencies
   displayName: 'Install dependencies'
+  workingDirectory: $(Build.ArtifactStagingDirectory)
+- script: c:\vcpkg\vcpkg.exe integrate install
+  displayName: 'Integrate vcpkg'
-- 
2.55.0

